packages/public-api/src/systemApiRouter.ts

```diff
@@ -9,8 +9,7 @@ systemApiRouter.post('/scan', async (req, res, next) => {
     const scanReport = systemApi.apiScanReportSchema.parse(req.body);
     await syncWebPageScanResult(scanReport);
   } catch (e) {
-    // TODO: report to rollbar
-    console.log(e);
+    logger.error(e);
```

Comment:
This is a fall-through error handler, see the `next(e)` line. Not sure why would we explicitly need to track it here.